src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+$main-color: #00acdc;
+
+.card {
+  height: 408px;
+  width: 200px;
+  color: #060B35;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto;
+    margin-bottom: 40px;
```

Comment:
You can merge the two margin properties into one by writing 'margin: 0 auto 40px;'